js/timer.js

```diff
@@ -1,13 +1,16 @@
 const config = require('../config/config.json')
 const text = require(`../config/text_${config.lang}.json`).timer
 let current_timers = []
-
+const seconds_per_loop = config.seconds_per_loop
```

Comment:
du müsstest noch config-template.json anpassen mit allen Variablen, die du in config.json hinzugefügt hast.